### PR TITLE
add amp-video example

### PR DESCRIPTION
**Pull request by ymotongpoo**
this p-r adds amp-video html sample and assets required in the file. Slightly modified gulpfile.js to copy video files.

### CODE REVIEW

File: src/amp-video.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+
+  <!-- #### Basic Usage -->
+  <!--
+    A simple video with controller. Boolean attribute (_controls_ in this case) must be without values.
+    Expression with value like `contorls="controls"` is invalid.
```
Comment (adewale) [inline src/amp-video.html]: Change to:
Expression with value like `controls="controls"` is invalid.

File: src/amp-video.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<!--
+  #### Introduction
+
+  Embed videos into your AMP HTML files.
```
Comment (sebastianbenz) [inline src/amp-video.html]: Should we add a comment that videos need to be served via https?.

**Comment by sebastianbenz** [comment]
Lgtm if you fix the typo.

**Comment by ymotongpoo** [comment]
fixed typo and added HTTPS notice in Introduction. PTAL.

**Pull request merged**